docs/database/oracledb.md

```diff
@@ -0,0 +1,110 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Oracle Database
+--->
+
+# Semantic conventions for Oracle Database
+
+**Status**: [Release Candidate][DocumentStatus]
+
+<!-- toc -->
+
+- [Spans](#spans)
+- [Metrics](#metrics)
+
+<!-- tocstop -->
+
+The Semantic Conventions for *Oracle Database* extend and override the [Database Semantic Conventions](README.md).
+
+## Spans
+
+Spans representing calls to Oracle Database adhere to the general [Semantic Conventions for Database Client Spans](database-spans.md).
+
+`db.system.name` MUST be set to `"oracle.db"` and SHOULD be provided **at span creation time**.
+
+The following table outlines the span attributes applicable to Oracle Database.
+
+<!-- semconv span.db.oracledb.client -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`db.namespace`](/docs/attributes-registry/db.md) | string | The database associated with the connection, qualified by the instance name, database name and service name. [1] | `ORCL1|PDB1|db_high.adb.oraclecloud.com`; `ORCL1|DB1|db_low.adb.oraclecloud.com` | `Conditionally Required` If available without an additional network call. | ![Release Candidate](https://img.shields.io/badge/-rc-mediumorchid) |
```

Comment:
I would expect the domain name to be in `server.address` and `server.port`, is there a reason to put it in `db.namespace`?

Reply:
thanks! so the service name is usually not the same as domain name? Does it usually follow FQDN pattern? it makes sense.

 If it's not super-common to use FQDN for service-name, would it be possible to update an example?

Reply:
Yes the service name need not follow FQDN pattern. I have added an example without FQDN:

`"ORCL1|DB1|order-processing-service"`